temporalio/exceptions.py

```diff
@@ -136,6 +136,20 @@ def details(self) -> Sequence[Any]:
         return self._details
 
 
+class ActivityPausedError(FailureError):
```

Comment:
This is not really a failure error. A failure error is an error modeled in proto in https://github.com/temporalio/api/blob/master/temporal/api/failure/v1/message.proto and catchable on the workflow side. I think this may need to be an error in the `activity` module. I also don't think it has details.